pkgs/applications/system/coolercontrol/coolercontrol-ui-data.nix

```diff
@@ -11,7 +11,7 @@ buildNpmPackage {
   inherit version src;
   sourceRoot = "${src.name}/coolercontrol-ui";
 
-  npmDepsHash = "sha256-t+QShKaXpQuEzeeu/ljBBEzeYsxqvMpx5waDZ2gyPAI=";
+  npmDepsHash = "sha256-Y1548pXJKByQ42UNsz8Opiui/tXcnfMhkshlZMTMC4I=";
 
   preBuild = ''
     autoPatchelf node_modules/sass-embedded-linux-x64/dart-sass/src/dart
```

Comment:
This patch is no longer needed. 
(This embedded dependency has been replaced with a non-binary alternative) 
So you can probably remove all this autoPatchelf stuff.